exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if(mark >= 80){
+    return "A";
+  }else if(mark < 80 && mark > 60) {
```

Comment:
There's actually no need to check that `mark < 80` here -- if it were, the first statement would have evaluated to true and returned.